tests/installation/installation_overview.pm

```diff
@@ -26,6 +26,12 @@ sub run() {
     # preserve it for the video
     wait_idle 10;
 
+    # poo#11438 - Low space warning for before migration
+    if (check_var("UPGRADE", "LOW_SPACE")) {
```

Comment:
I do not like checking for something that should not be there. For example if fonts change needle will be obsolete and nobody will notice.
I will check that there is no warning messages in packages section.